src/language-server/providers/schema/engine.ts

```diff
@@ -47,12 +50,13 @@ export class EngineSchemaProvider implements GraphQLSchemaProvider {
       );
     }
 
-    const { data, errors } = await this.client.execute<GetSchemaByTagQuery>({
+    const { data, errors } = await this.client.client.query({
       query: SCHEMA_QUERY,
       variables: {
         id: this.config.graph,
         tag: override && override.tag ? override.tag : this.config.variant,
       },
+      fetchPolicy: "no-cache",
     });
     if (errors) {
       // XXX better error handling of GraphQL errors
```

Comment:
Yeah, I agree. But see the first comment - they are close enough to the state they were in before now.